app/index.js

```diff
@@ -81,6 +81,7 @@ module.exports = class GeneratorLicense extends Generator {
         name: 'license',
         message: 'Which license do you want to use?',
         default: this.options.defaultLicense,
+        when: this.options.license == null,
```

Comment:
I've tried `when: this.options.license == null || !(this.options.license in licenses),` but it doesn't seem to work, any pointers?